cpp/misra/src/codeql-suites/misra-default.qls

```diff
@@ -1,10 +1,2 @@
-- description: MISRA C++ 2023 (Default)
-- qlpack: codeql/misra-cpp-coding-standards
-- include:
-    kind:
-    - problem
-    - path-problem
-- exclude:
-    tags contain:
-    - external/misra/audit
-    - external/misra/default-disabled
+- description: "DEPRECATED - MISRA C++ 2023 - use misra-cpp-default.qls instead"
+- import: codeql-suites/misra-cpp-default.qls
```

Comment:
whitespace nit, no trailing newline, if that matters.

Reply:
I don't think it makes a difference, but a trailing newline is consistent with the other suites so I'll add it.